lib/elastomer/client/bulk.rb

```diff
@@ -174,7 +174,7 @@ def initialize( client, params = {} )
         @actions = []
         @current_request_size = 0
         @current_action_count = 0
-        self.request_size = params.delete(:request_size)
+        self.request_size = params.delete(:request_size) || 10 * 1024 * 1024
```

Comment:
I would recommend changing this magic number to a constant and then use the constant here.

```ruby
class Bulk
  DEFAULT_REQUEST_SIZE = 10 * 2**20  # 10 MB
  ...
end
```